intelmq/bots/BOTS

```diff
@@ -826,6 +826,18 @@
                 "port": "<port>"
             }
         },
+        "UDP": {
+            "description": "UDP is a simple UDP bot responsible to send events to a udp port (e.g.: syslog daemon).",
+            "module": "intelmq.bots.outputs.udp.output",
+            "parameters": {
+                "field_delimiter": "|",
+                "format": "<json/delimited> read README.md",
+                "header": "<header text>",
+                "ip": "<ip>",
```

Comment:
not implicitly clear what this is. Is this the ip address of the UDP desitnation server?
If so, I suggest nameing it ``destination_server`` or ``udp_server`` or similar. Does it need to be an IP address? Can't it be a hostname?

I am a big fan of naming this so that they are very clear what they imply. even without reading the README.md